cmake/FindnetCDFCxx.cmake

```
@@ -99,10 +99,18 @@ endif()
 mark_as_advanced(netCDF_CXX_LIBRARY)
 
 bout_inspect_netcdf_config(_ncxx4_version "${NCXX4_CONFIG}" "--version")
-string(REGEX REPLACE "netCDF-cxx4 \([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)" "\\1" _netcdfcxx_version_major "${_ncxx4_version}")
-string(REGEX REPLACE "netCDF-cxx4 \([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)" "\\2" _netcdfcxx_version_minor "${_ncxx4_version}")
-string(REGEX REPLACE "netCDF-cxx4 \([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)" "\\3" _netcdfcxx_version_patch "${_ncxx4_version}")
-set(netCDFCxx_VERSION "${_netcdf_version_major}.${_netcdf_version_minor}.${_netcdf_version_patch}${_netcdf_version_note}")
+if (${_ncxx4_version})
+  string(REGEX REPLACE "netCDF-cxx4 \([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)" "\\1" _netcdfcxx_version_major "${_ncxx4_version}")
+  string(REGEX REPLACE "netCDF-cxx4 \([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)" "\\2" _netcdfcxx_version_minor "${_ncxx4_version}")
+  string(REGEX REPLACE "netCDF-cxx4 \([0-9]+\)\.\([0-9]+\)\.\([0-9]+\)" "\\3" _netcdfcxx_version_patch "${_ncxx4_version}")
+
+  message("${_ncxx4_version}")
```

Just to add a bit more context to this output:

```suggestion
  message(STATUS "Found netCDFCxx version ${_ncxx4_version}")
```